Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -235,6 +239,13 @@ namespace WinMLRunnerTest
             std::string copyCommand = "rd /s /q ";
             copyCommand += std::string(INPUT_FOLDER_PATH.begin(), INPUT_FOLDER_PATH.end());
             system(copyCommand.c_str());
+            try
```

Comment:
There is already code to remove tensor output files on line 259. It would be redundant to have both class and method clean up functionality for tensor output files.

Class clean up would be more efficient but it doesn't catch the case if the user runs only one test. Thoughts?

Reply:
Confirmed that class only cleanup only runs when you run all tests.  I actually think this is better... been very frurstrated trying to debug, while single test runs (failed ones specifically), get deleted.  You don't _have_ to delete like you do without output.csv, since the folder names are unique per test.  I think it would be best to only cleanup at the class level.

Eventually, might want to improve the rest of the tests to leverage the same directory structure capability, so output.csv can exist and the tests can be run in parallel (I don't want to own that change with this change though... maybe another issue)?

Reply:
Thanks for the change to make folder names unique per test! However, I'm running into the issue where if I run the same test twice without cleanup, the previous run(s) append to the same output tensor file and contaminates the test result. Thoughts on how to address this?

Reply:
Would need to add a CleanupMethod to delete the specific test data before executing... or re-add the Cleanup per method, which would only delete the single test directory... hmmm